app/uk/gov/hmrc/tariffclassificationfrontend/controllers/SuspendCaseController.scala

```diff
@@ -41,7 +41,7 @@ class SuspendCaseController @Inject()(verify: RequestActions,
 
   override protected val config: AppConfig = appConfig
   override protected val caseService: CasesService = casesService
-  private val form: Form[String] = AddNoteForm.form
+  private val form: Form[String] = AddNoteForm.getForm("error.empty.suspend.note")
```

Comment:
Seems a bit of a strange thing to be passing into a `getForm` method? maybe just pass in `suspend` and the `error.empty.{x}.note` key is generated inside the form?

Reply:
updated as suggested